spec/presenters/sufia/collection_presenter_spec.rb

```diff
@@ -58,4 +58,9 @@
     subject { presenter.total_items }
     it { is_expected.to eq 0 }
   end
+
+  describe "#identifier" do
+    subject { presenter.identifier }
```

Comment:
@jeremyf it's nil because the sample collection's identifier it nil. This is what it looks like if you just create a collection without filling in anything for the identifier.